dronecan/sensors/rc/1141.RCLinkStats.uavcan

```diff
@@ -0,0 +1,39 @@
+#
+# This definition describes metadata and statistics for a RF control link.
+# It provides information regarding a link's signal quality, as well as transmitter
+# power, and antenna information. It accomodates single-radio, and dual-radio
+# link architectures. It provides information for both uplink (control unit to airborne
+# receiver), and downlink (airborne receiver to control unit) data.
+#
+
+uint8 STATUS_RSSI_VALID = 1 # RSSI field is valid
+
+# Uplink - received signal strength, in decibel-milliwatts (RSSI dBm) for antennas 1 and 2 respectively. 
+# Ranges from # -255 to 0. (An integer value of 100 means RSSI is -100dBm)
+uint8 uplink_rssi_1
+uint8 uplink_rssi_2
+
+# Link quality, on a scale from 1 to 255. Is a proxy for packet loss. 0 means data is unavailable. 1 and 255 are mapped to 0% and 100% of packets received.
+uint8 uplink_link_quality
+
+# Signal-to-noise ratio.
+int8 uplink_snr
+
+# This is an implementation-specific enum. May represent the OTA protocol used,
+# and transmission rate.
+uint8 rf_mode
+
+# This is an implementation-specific enum describing the current transmitter nominal
+# power level.
+uint8 uplink_tx_power
+
+# Statistics for downlink data, ie telemetry. See their respective `uplink` fields for
+# descriptions.
+uint8 downlink_rssi
+uint8 downlink_link_quality
+int8 downlink_snr
+
+# Active antenna on transmitter and receiver. Example: 1 for antenna 1, and 2 for antenna 2. 
+# Not applicable for single-antenna links.
+uint8 tx_active_antenna
+uint8 rx_active_antenna
```

Comment:
Is this the uplink or the downlink? Or is this a typo - ie. do you mean `uplink_tx_active_antenna` and `uplink_rx_active_antenna` or something else?

From discussion on MAVLink issue:

> a unit which has two antenna may or may not split them into one is for Tx and the other one for Rx.
Usually, at least in nearly all cases I'm aware of, they both would be used to e.g. receive, which is when called receive diversity. Note that many bidirectional units are half-duplex, i.e. work as transmit-receive-transmit-receive-.... so that both antenna can be used for receiving one frame at the same time.
I guess the system you have in mind is a system there two over-the-air channels/frequencies are used in parallel, in a full-duplex sense, there one channel/frequency is for transmission in one direction and the other for transmission in the other direction. As said, essentially all systems I'm aware of are of the half-duplex nature.
Anyway, also the full-duplex case can be represented by the suggested fields.

My assumption is therefore that you need to be able to get info for antenna usage on both uplink and downlink.
(Note, Ollie uses Rx for air unit, and Tx for ground unit).

I'm still confused by the intent here - i.e. if you have bidirectional data, and the antenna both receive and both send in a cycle how would you represent it. I mean you'd have to show "no rx antenna and both 1 and 2 on the tx, then visa versa). 
Also if it is switching, how useful is that?